pkgs/top-level/all-packages.nix

```diff
@@ -31082,8 +31082,6 @@ with pkgs;
 
   cryptop = python3.pkgs.callPackage ../applications/blockchains/cryptop { };
 
-  dashpay = callPackage ../applications/blockchains/dashpay { };
```

Comment:
Please also remove this directory.

Reply:
Oh, forgot to add that to the commit. Fixed.